dask_saturn/core.py

```diff
@@ -312,9 +312,19 @@ def _start(
     def register_default_plugin(self):
         """Register the default SaturnSetup plugin to all workers."""
         log.info("Registering default plugins")
+        outputs = {}
         with Client(self) as client:
             output = client.register_worker_plugin(SaturnSetup())
-            log.info(output)
+            outputs.update(output)
+        output_statuses = [v["status"] for v in outputs.values()]
+        if "OK" in output_statuses:
```

Comment:
I don't understand this so I'm not sure how to review it, sorry. Could you show me an example of what an `output`  dict looks like? I didn't understand if the image in the PR description showed the expected logs **as of this PR** or the logs **without this PR** that you'd like to make shorter.

Reply:
Does the context of the linked ticket not provide this?

Reply:
oh! totally missed that, sorry. I'm not used to there being tickets linked for these public repos. That describes it perfectly, thank you.

Reply:
No worries, the output is like: `{'tcp://10.0.0.84:33237': {'status': 'OK'}}`